# Module 12: Inference About Two Population Means

In this module, we discuss how to test whether two populations have the same mean. We expand upon the matched pairs T-test from Module 11 to allow for independent samples from the two populations. We also show how to find a confidence interval for the difference between these two populations' means.

The null hypothesis that we will be testing in this module is that the means of our two populations are the same. An equivalent way to say this is that the difference between the two population means is zero. The alternatives that we consider are that the difference between these two means is less than zero, greater than zero or not equal to zero. 

Saying that the difference is less than zero depends on the order of the means when we subtract them. Therefore, we have to be careful when specifying the order of our means in R functions to make sure that our alternative hypothesis is saying what we think it is.

The first T-test that we look at in this module is simply called the two-sample T-test. This test does not require that our data come in pairs, making it applicable in many more situations than the matched pairs T-test from Module 11. The two-sample T-test also makes no assumptions about the variances of our two populations. That is, we allow the populations' variances to be different. 

We carry out the two-sample T-test in R using the "t.test()" function. We will use many of the same inputs that we discussed in Module 11, so check back there for more details. In particular, we use "x", "y", "alternative" and "conf.level". We are now working with unpaired data, so we could set "paired" to "FALSE". The default value of "paired" is "FALSE" however, so we can just omit this input.

The following code cell has independent samples from two different populations. Let's use a two-sample T-test to test the null hypothesis that these populations have the same mean versus the alternative hypothesis that their means are dbifferent. 

In [1]:
x1 = c(0.98, 1.32, 1.33, 0.47, 0.23, 1.90, 0.71, 0.70, 1.80, 1.86)
x2 = c(0.33, 0.63, 2.60, 1.42, 0.20, 1.45, 2.88, 2.97, 1.81, 2.65)
t.test(x=x1, y=x2)


	Welch Two Sample t-test

data:  x1 and x2
t = -1.4604, df = 14.27, p-value = 0.1658
alternative hypothesis: true difference in means is not equal to 0
95 percent confidence interval:
 -1.3908133  0.2628133
sample estimates:
mean of x mean of y 
    1.130     1.694 


The p-value from this test is 0.1658, so we cannot conclude that the means of these groups are different.

We were able to omit the "alternative" input because our alternative hypothesis is two-sided, the default value.

The following code cell has independent samples from two new populations. Let's test the null hypothesis that their means are equal against the alternative hypothesis that the mean of the first population is larger. We will also get a $99\%$ confidence interval for the difference between the populations' means.

In [2]:
x1 = c(106, 103, 114, 95, 102, 107, 105)
x2 = c(99, 102, 100, 98, 98)
t.test(x=x1, y=x2, alternative="greater")
t.test(x=x1, y=x2, conf.level=0.99)


	Welch Two Sample t-test

data:  x1 and x2
t = 2.2532, df = 7.3563, p-value = 0.02856
alternative hypothesis: true difference in means is greater than 0
95 percent confidence interval:
 0.8546567       Inf
sample estimates:
mean of x mean of y 
 104.5714   99.4000 



	Welch Two Sample t-test

data:  x1 and x2
t = 2.2532, df = 7.3563, p-value = 0.05712
alternative hypothesis: true difference in means is not equal to 0
99 percent confidence interval:
 -2.729444 13.072301
sample estimates:
mean of x mean of y 
 104.5714   99.4000 


Notice that we ran the "t.test()" function twice. The first time, with "alternative="greater"", tests the appropriate hypotheses. The second time, with "conf.leve=0.99", gives the correct confidence interval. We cannot get all the information that we want for this question from a single run of the "t.test()" function.

The p-value from our test is 0.02856, so we reject our null hypothesis and conclude that the mean of population one is larger than the mean of population two. Our $99\%$ confidence interval for the difference between these two means is $(-2.729444, 13.072301)$. This confidence interval does contain our null hypothesis difference of zero, but the confidence interval is at a higher significance level than our hypothesis test (the hypothesis test would not have rejected our null hypothesis at $\alpha=0.01$).

We had to be careful when specifying the direction of our alternative hypothesis. The difference is always calculated as the mean of population one minus the mean of population two. That is, the mean of the population corresponding to "x" minus the mean of the population corresponding to "y". 